integration-test/src/main/java/com/sequenceiq/it/cloudbreak/actor/CloudbreakActor.java

```diff
@@ -47,6 +47,11 @@ public CloudbreakUser create(String tenantName, String username) {
         return new CloudbreakUser(accessKey, secretKey, username + " at tenant " + tenantName);
     }
 
+    @Override
+    public CloudbreakUser createInternal(String tenantName) {
```

Comment:
Internalactor's has nothing to do with tenant. It's always:
crn:altus:iam:us-west-1:altus:user:__internal__actor__

Reply:
What is the reason you need to have internalActor?

Reply:
I wanted to ask the same. You can use original user and use internal client to call this new endpoint internally

Reply:
It just remained there from the previous InternalOnly authorization, currently it is only used here: https://github.com/hortonworks/cloudbreak/blob/6c28b0a9a04223dc19567b5f20858a43a0281574/integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/FreeIpaSyncTest.java#L50

Reply:
I see, it is a completely different test, not related to the new endpoints. Then I can accept this, but anyway I would prefer to using internal client instead of creating an internal cloudbreakuser cc @aszegedi

Reply:
I removed it completely and used the internal client